### DISCUSSION TITLE
询问：对 cdn.jsdelivr.net 的处理

### DISCUSSION THREAD

**Discussion by 0xffffharry** [v2fly/domain-list-community#872] [General] [answered]
需不需要将 cdn.jsdelivr.net 从 cn 类中移除

jsdelivr  类中有这样一行：
`full:cdn.jsdelivr.net @cn`

**Comment by IceCodeNew** [comment]
我的意见是暂时不必。
1. 他们已经把中国地区的请求转到了“近中国”节点上，直连质量应该可以期待——也即暂时不处理也不应该影响用到这条规则的国内用户访问
2. 未来有恢复 ICP 资格的可能

先观察吧。

**Comment by if1y** [comment]
jsdelivr已将中国地区线路切至Cloudflare (https://github.com/jsdelivr/jsdelivr/issues/18367)

**Comment by IceCodeNew** [comment]
> jsdelivr已将中国地区线路切至Cloudflare ([jsdelivr/jsdelivr#18367](https://github.com/jsdelivr/jsdelivr/issues/18367))

为啥我读完觉得他们还在选择线路中……

**Comment by 0xffffharry** [comment]
换成cloudflare还不如原来的fastly呢

**Comment by 107295472** [comment]
不行啊，cdn.jsdelivr.net不能访问

**Comment by 107295472** [comment]
怎么走代现啊，full:cdn.jsdelivr.net  写到bat里了，改不了

**Comment by 107295472** [comment]
cdn.jsdelivr.net 走代理可以吗，这个访问不了

**Comment by IceCodeNew** [comment, marked as answer]
> cdn.jsdelivr.net 走代理可以吗，这个访问不了

你的问题不是在别的 issue 里得到解答了吗？无法访问的只有你家的移动网络对不对，那这个项目就没有可以帮助你的地方。
你可以自己 fork 这个项目然后做你想要的修改。

不要继续在这个 issue 上面提你的问题了。搜索相关的 issue 是好事，但是这个 issue 希望解决的事和你自己的网络问题是两码事。你可以在你自己提的 issue 上寻求建议。
